packages/x-data-grid/src/hooks/features/editing/getDefaultCellValue.ts

```diff
@@ -0,0 +1,17 @@
+import { GridColDef } from '../../../models/colDef/gridColDef';
+
+export const getDefaultCellValue = (colDef: GridColDef): '' | null | undefined | false => {
```

Comment:
Can't typescript infer the return type?
```suggestion
export const getDefaultCellValue = (colDef: GridColDef) => {
```